internal/cli/help.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/gruntwork-io/terragrunt/internal/errors"
 	"github.com/urfave/cli/v2"
+	"golang.org/x/exp/maps"
```

Comment:
I don't think we need to use this anymore:
https://github.com/gruntwork-io/terragrunt/pull/3881

Reply:
Thanks.